public/data/_list.json

```diff
@@ -2,7 +2,7 @@
   { "username": "eddiejaoude", "avatar": "https://github.com/eddiejaoude.png" },
   {
     "username": "nhcarrigan",
-    "avatar": "https://avatars.githubusercontent.com/u/63889819?v=4"
+    "avatar": "https://github.com/nhcarrigan.png"
```

Comment:
I think the PR cant get merged, because the _list.json gets autogenerated now. The script reads the avatar-url from the corresponding json file in `public/data`

Reply:
It probably needs to be resolved in that file as well
[link](https://github.com/EddieHubCommunity/LinkFree/blob/352a98889ba2329eb1ee03927fb9e46f9a145ca7/public/data/nhcarrigan.json#L4)